resources/klusterlet/klusterlet.yaml

```diff
@@ -5,10 +5,10 @@ kind: Klusterlet
 metadata:
   name: klusterlet
 spec:
-  registrationImagePullSpec: {{ .RegistrationImageName }}
-  workImagePullSpec: {{ .WorkImageName }}
-  clusterName: {{ .ManagedClusterNamespace }}
-  namespace: {{ .KlusterletNamespace }}
+  registrationImagePullSpec: "{{ .RegistrationImageName }}"
+  workImagePullSpec: "{{ .WorkImageName }}"
```

Comment:
do we need quotes for images ?

Reply:
I could probably pull those out if there is not much of them having weird values

Reply:
changed